exasol/analytics/audit/audit.py

```diff
@@ -0,0 +1,111 @@
+from inspect import cleandoc
+from typing import List
+
+from exasol.analytics.query_handler.query.select import (
+    DB_OBJECT_NAME_TAG,
+    DB_OBJECT_SCHEMA_TAG,
+    DB_OBJECT_TYPE_TAG,
+    DB_OPERATION_TYPE_TAG,
+    AuditQuery,
+    ModifyQuery,
+    SelectQueryWithColumnDefinition,
+)
+from exasol.analytics.schema import (
+    Column,
+    SchemaName,
+    TableLikeName,
+    TableLikeNameImpl,
+    decimal_column,
+    timestamp_column,
+    varchar_column,
+)
+
+
+class TableDescription:
```

Comment:
This class describes an SQL table by its attributes 
* `table` (instance of (`TableLikeName`)) defining the name and optionally the database schema of the SQL table 
* `columns` the names and types of the columns of the SQL table